tests/unit/test_slide.py

```diff
@@ -384,15 +384,22 @@ def it_constructs_from_args(self, request):
         _slides_path = "/foo/bar/"
         _processed_path = "/foo/bar/wsislides/processed"
         _valid_extensions = [".svs", ".tiff"]
+        _keep_slides = ["mywsi.svs"]
 
-        slideset = SlideSet(_slides_path, _processed_path, _valid_extensions)
+        slideset = SlideSet(
+            _slides_path, _processed_path, _valid_extensions, keep_slides=_keep_slides
```

Comment:
```suggestion
            _slides_path, _processed_path, _valid_extensions, _keep_slides
```
I would stick for the same arguments style (positional) for all of them ;)